executor/executor.go

```diff
@@ -226,7 +226,7 @@ func (e *CancelDDLJobsExec) NextChunk(ctx context.Context, chk *chunk.Chunk) err
 	}
 	numCurBatch := mathutil.Min(e.maxChunkSize, len(e.jobIDs)-e.cursor)
 	for i := e.cursor; i < e.cursor+numCurBatch; i++ {
-		chk.AppendInt64(0, e.jobIDs[i])
+		chk.AppendString(0, fmt.Sprintf("%d", e.jobIDs[i]))
```

Comment:
Why made this change?

Reply:
https://github.com/pingcap/tidb/blob/master/plan/planbuilder.go#L653-L659
can explain.